lib/cisco_node_utils/cmd_ref/vrf.yaml

```diff
@@ -64,8 +64,27 @@ route_distinguisher:
   _exclude: [ios_xr]
   get_value: '/^rd (\S+)$/'
   set_value: '<state> rd <rd>'
+  context: ['vrf context <vrf>']
```

Comment:
@glennmatthews 
While I agree that splitting the yaml would be a good idea, can we create a story for it and put it in the backlog?  I'd rather not make a change like that in the grpc branch, which will just make future merges more difficult.